src/main/java/isaac/bastion/utils/BastionSettingManager.java

```diff
@@ -23,18 +24,22 @@ public BastionSettingManager() {
 	}
 
 	private void initSettings() {
-		MenuSection menu = PlayerSettingAPI.getMainMenu().createMenuSection("Bastion", "All settings related to Bastion");
+		ConfigurationSection config = Bastion.getPlugin().getConfig().getConfigurationSection("bastions");
+		String bastionType = config.getKeys(false).toArray(new String[0])[0];
```

Comment:
This is super hacky, you really shouldn't be relying on undocumented order of options in a YAML file and you shouldnt be parsing the config during runtime.
Use something like
```
BastionType.getBastionType(BastionType.getDefaultType(().getMaterial();

```